homeassistant/components/zwave/__init__.py

```diff
@@ -752,24 +823,166 @@ def start_zwave(_service_or_event):
     return True
 
 
+class ZWaveDeviceEntityValues():
```

Comment:
So I started this, but it will create more of a mess than it will solve. The logic to build the entity uses a lot of module-level information from zwave that would need to be moved out of place even further to avoid circular dependencies. For example, we use `DOMAIN`, `CONF_IGNORED`, `CONF_POLLING_INTENSITY`, `DATA_ZWAVE_DICT`, and the `object_id` helper.